test/specs/translate.spec.js

```diff
@@ -179,3 +179,176 @@ describe('Translate tests', () => {
   })
 
 })
+
+describe('Translate tests without Vue', () => {
```

Comment:
This is the same as the `describe()` above but with `Vue.config` replaced with a local `config` object.